### DISCUSSION TITLE
Athens Canvas

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#968] [General]
Discussion about the potential benefits, uses, and implementation of a canvas style view of blocks, pages and more within Athens.

inspiration:
- https://natto.dev/
- https://milanote.com/
- https://miro.com
- https://www.figma.com/
- https://whimsical.com

questions:
-

**Comment by juniusfree** [comment]
Adding this here for inspiration/discussion
https://subconscious.substack.com/p/thought-legos

**Reply by juniusfree** [reply]
https://whimsical.com

![image](https://user-images.githubusercontent.com/8164667/114796923-a3ab5e80-9dc4-11eb-9161-b624f79951d0.png)

**Reply by jefftangx** [reply]
Whimsical is a Clojure startup - https://whimsical.com/blog/how-we-built-whimsical

+ Figma for infinite canvas

**Reply by juniusfree** [reply]
https://www.figma.com/figjam/

**Reply by jefftangx** [reply]
@juniusfree this is huge! Gonna have to start ideating in Figjam @jsmorabito
